scanner/src/components/BasicInformation/index.svelte

```diff
@@ -61,7 +61,7 @@
       width: 5rem;
       height: 5rem;
       background: var(--background-logo);
-      border-radius: 0.313rem;
+      border-radius: 0.31rem;
```

Comment:
![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/09db9a19-4d24-4984-b713-52d301787202)
This does not seem to me like a sensible `border-radius` value